### PR TITLE
Add bylaws page to website 

### PR DESCRIPTION
**Pull request by PaulAngus**

### CODE REVIEW

**Comment by PaulAngus** [comment]
current page -> http://cloudstack.apache.org/bylaws.html

replacement version -> https://acs-www.shapeblue.com/www/bylaws.html

File: source/cloudstack-faq.html.markdown [hunk 1]
```diff
@@ -1,58 +0,0 @@
...
-
-<h1 id="indicators">Apache CloudStack: FAQ</h1>
-
-</div>
-
-</div>
-
-</div>
-
-This page is for Frequently Asked Questions (FAQ) having to do with Apache CloudStack as a project and its use.
```
Comment (rhtyd) [inline source/cloudstack-faq.html.markdown]: Removing this would cause 404 on http://cloudstack.apache.org/cloudstack-faq.html @PaulAngus ?

**Comment by rhtyd** [comment]
LGTM, content is still the same; only formating changes.

File: source/cloudstack-faq.html.markdown (hunk 1 above)
Comment (PaulAngus) [reply source/cloudstack-faq.html.markdown]: Have I missed a link to the page somewhere?  I've removed it from the menu.
IMO the page would actually detract from someone's view of CloudStack as it appears to be half started.

**Comment by PaulAngus** [comment]
Confirmed fix: 
image
is now:
image

**Pull request merged**